### DISCUSSION TITLE
Optimizing repository discoverability with Topics

### DISCUSSION THREAD

**Discussion by germainelee** [owncast/owncast#3854] [Ideas] [answered]
Hi,

The steady growth of [gaining stars](https://star-history.com/#Owncast/owncast&Date) on this project is a good indicator that people sees value in this project. The current list of Topics is great in helping people find this project, and I wonder if we could optimize discoverability by adding more topics since users of Owncast are familiar with GitHub and might have found this project via GitHub too. 

Below are some Topics I have in mind that self-host enthusiasts would search for besides the current "self-hosted" and "rtmp" topics:

- Decentralized _(2,092 public repositories matching this [topic](https://github.com/topics/decentralized))_
- ActivityPub _(mastodon, lemmy, peertube etc use this [topic](https://github.com/topics/activitypub))_
- Federation _(misskey, pixelfed, writefreely etc use this [topic](https://github.com/topics/federation))_
- Fediverse _(mastodon, lemmy, pixelfed etc use this [topic](https://github.com/topics/fediverse))_

Please feel free to share your thoughts and ideas. Thanks!

## What is topics?
Topics are the ones in the red box. Learn more at [GitHub Doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics)
![image](https://github.com/user-attachments/assets/dee79cc0-e55a-44eb-9f1a-421cc0c5cdc2)

**Comment by gabek** [comment, marked as answer]
Sure, I don't see why not. Decentralized and Federation are pretty vague since they are concepts, but I don't see any reason we can't add them.

**Reply by germainelee** [reply]
Awesome! Contributors don't have access to edit the tags though lol

**Reply by gabek** [reply]
I went ahead and added them just now.
